cap/worker_options.go

```diff
@@ -0,0 +1,120 @@
+package cap
+
+import (
+	"github.com/capatazlib/go-capataz/internal/c"
+)
+
+// Restart specifies when a goroutine gets restarted
+type Restart = c.Restart
+
+// Permanent specifies that a goroutine should be restarted whether or not there
+// are errors.
+//
```

Comment:
On the silly side, for the workflow system that built forever ago, we used the terms `evergreen`, `deciduous`. and `annual`. 🙄